CHANGELOG.md

```diff
@@ -54,26 +54,80 @@ Classify the change according to the following categories:
     - Added **replacements_future_cost_after_tax**
     - Added **replacements_present_cost_after_tax**
     - Added **offgrid_microgrid_lcoe_dollars_per_kwh**
-    - Changed **lifecycle_capital_costs_plus_om** and **lifecycle_om_costs_bau** field names to include before/after tax
+    - Changed **lifecycle_capital_costs_plus_om** field name to **lifecycle_capital_costs_plus_om_after_tax**
+    - Changed **lifecycle_om_costs_bau** field name to **lifecycle_om_costs_before_tax_bau**
+- **ElectricLoadInputs**
+    - Removed default value for **critical_load_met_pct**. If user does not provide this value, it is defaulted depending on **Settings -> off_grid_flag**
+    - Added **operating_reserve_required_pct**
```

Comment:
this was added to WindInputs not ElectricLoadInputs

Reply:
This was retroactively updated for the previous PR, in which `operating_reserve_required_pct` was in fact added to `ElectricLoadInputs`. I've updated the CHANGELOG for this PR to include `operating_reserve_required_pct` for WindInputs